backend/graphql/schema.graphql

```diff
@@ -21,7 +21,11 @@ type Query {
     "Return pre-defined user roles in the application"
     role(id: ID @eq): Role @find
 
+    "Return a permission by ID"
     permission(id: ID @eq): Permission @find
+
```

Comment:
Should this be a separate query or should the search parameter be added to the existing users query?

Reply:
Good question. I suppose if I had the option, I would add it to the existing users query, but I have not looked into how to combine two different directives (`eq` and `@search`) on the same query.

Reply:
users, not user

I think searchUsers is probably a better name than users. Perhaps this should just replace the users endpoint.

Reply:
That's a good suggestion. I'll refactor.